eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/producer/CloudEventProducer.java

```diff
@@ -75,7 +76,7 @@ private void validateCloudEvent(final CloudEvent cloudEvent) {
 
     private RequestParam buildCommonPostParam(final CloudEvent cloudEvent) {
         validateCloudEvent(cloudEvent);
-        final byte[] bodyByte = EventFormatProvider.getInstance().resolveFormat(cloudEvent.getDataContentType())
+        final byte[] bodyByte = Objects.requireNonNull(EventFormatProvider.getInstance().resolveFormat(cloudEvent.getDataContentType()))
             .serialize(cloudEvent);
         final String content = new String(bodyByte, StandardCharsets.UTF_8);
```

Comment:
how about this place change to 
```java
final String content = new String(bodyByte, Constants.DEFAULT_CHARSET);
```

Reply:
My IDE is not recogninzing this Constants.DEFAULT_CHARSET. Can you put the import as well?